src/components/Home.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { showPlanet } from '../redux/home/homeSlice';
+import logo from '../assets/solar-system.png';
+import earth from '../assets/worldwide.png';
+import venus from '../assets/venus.png';
+import mars from '../assets/mars.png';
+import mercury from '../assets/mercury.png';
+import saturn from '../assets/saturn.png';
+import neptune from '../assets/neptune.png';
+import jupiter from '../assets/jupiter.png';
+import uranus from '../assets/uranus.png';
+
+const Home = () => {
+  const planets = useSelector((state) => state.planets.planets);
+
+  const dispatch = useDispatch();
+  const [searchInput, setSearchInput] = useState('');
+
+  const handleChange = (e) => {
+    e.preventDefault();
+    setSearchInput(e.target.value);
+  };
+
+  const filter = planets.filter((planet) => planet.name.toLowerCase()
+    .match(searchInput.toLowerCase()));
+
+  return (
+    <div>
+      {planets.length === 0 ? (
+        <p className="loader-container" />
+      ) : (
+        <div>
+          {planets && (
+            <>
+              <div className="header">
+                <div />
+                <h3>Number of Moons</h3>
+                <input
+                  onChange={handleChange}
+                  value={searchInput}
+                  placeholder="Search Planet"
+                />
+              </div>
+
+              <div className="background">
+                <img className="backImage" alt="icon" src={logo} />
+                <div className="title">
+                  <h1>SOLAR SYSTEM</h1>
+                  <h2>206 moons</h2>
```

Comment:
- [Optional] 206 is manually displayed as compared to what's done on the detail page :smile: